fabric/state.py

```diff
@@ -224,6 +224,13 @@ def _rc_path():
         help="default to parallel execution method"
     ),
 
+    make_option('--quit-parallel-run-on-errors',
```

Comment:
I'd say have the flag be the same name as the var, so `--parallel-exit-on-errors`